Source/WebCore/platform/graphics/cg/ImageDecoderCG.cpp

```diff
@@ -302,10 +302,14 @@ EncodedDataStatus ImageDecoderCG::encodedDataStatus() const
     if (uti.isEmpty())
         return EncodedDataStatus::Unknown;
 
+#if HAVE(CGIMAGESOURCE_WITH_SET_ALLOWABLE_TYPES)
+    RELEASE_ASSERT(isSupportedImageType(uti));
```

Comment:
Do we want to call `isSupportedImageType()` twice in a row just for this assertion? Maybe just add an RELEASE_ASSERT_NOT_REACHED inside the `if` body when `HAVE(CGIMAGESOURCE_WITH_SET_ALLOWABLE_TYPES`?